libs/community/langchain_community/vectorstores/azure_sql_server.py

```diff
@@ -0,0 +1,176 @@
+from typing import Any, Iterable, List, Optional
+
+from langchain_core.documents import Document
+from langchain_core.embeddings import Embeddings
+from langchain_core.vectorstores import VST, VectorStore
+from sqlalchemy import Column, Uuid, bindparam, create_engine, text
+from sqlalchemy.dialects.mssql import JSON, NVARCHAR, VARBINARY
+from sqlalchemy.engine import Connection, Engine
+from sqlalchemy.exc import DBAPIError
+from sqlalchemy.orm import Session
+
+try:
+    from sqlalchemy.orm import declarative_base
+except ImportError:
+    from sqlalchemy.ext.declarative import declarative_base
+
+import json
+import logging
+import uuid
+
+Base = declarative_base()  # type: Any
+
+_embedding_store: Any = None
+
+
+class AzureSQLServer_VectorStore(VectorStore):
```

Comment:
nit: name it `AzureSQL`. Check with Davide/Pooja on exact name. 
SQL Server is on premises product - whereas Azure SQL DB is cloud. we may want to convey it is available both on premises and cloud.

Reply:
Will you be changing the name to `SQLServer` as decided?

Reply:
Yes.

Reply:
Updated in the latest iteration.